deploy/50_ocm-agent-operator.Deployment.yaml

```diff
@@ -13,6 +14,16 @@ spec:
         name: ocm-agent-operator
     spec:
       serviceAccountName: ocm-agent-operator
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Am wondering if `preferredDuringSchedulingIgnoredDuringExecution` is a better option for node affinity here? Reason being, in past we came across some bugs in Monitoring where the infra nodes faced resource crunch and didn't leave any room for the pods to be scheduled during machine replacements or upgrades etc. Though yes, there are chances that this pod could be scheduled on worker nodes then and deprives the customer pods from using those resources instead. IMO, that might be tolerable than having OAO pod in pending state for some reason due to strict requirements. WDYT?

Reply:
A quick look at other on-cluster SRE operators shows that they're all `preferred` too. I have switched us from `required` to `preferred` in the latest commit. :+1: